cuebot/src/main/java/com/imageworks/spcue/dao/postgres/FrameDaoJdbc.java

```diff
@@ -102,7 +102,7 @@ public boolean updateFrameStopped(FrameInterface frame, FrameState state,
         "WHERE " +
             "frame.pk_frame = ? " +
         "AND " +
-            "(frame.str_state = ? OR frame.str_state = ?) " +
```

Comment:
Removing `TERMINATING` here because it is incorrectly accepting a frame that is not `RUNNING` (it has been killed).  The logic that `stopFrame` guards should only be executed if the frame was skill running.